form/index.php

```diff
@@ -0,0 +1,43 @@
+<?php
+//configurations
+include('Mail.php'); 
```

Comment:
Inconsistent Naming: Ensure you are consistent with your naming convention. The first includes starts with Uppercase and the second starts with lowercase. Kindly choose one (preferably the lowercase version) and stick with it.

Reply:
The `include('Mail.php')` comes from an inbuilt PEAR class installed on the server.
[https://www.qservers.net/process/knowledgebase.php?action=displayarticle&id=52](url)